.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,25 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
Something weird going on here... none of these files have anything to do with .travis.yml ...
And I'm sure you didn't mean to commit any wheels in here!